WooCommerce/Classes/Authentication/Epilogue/StorePickerViewModel.swift

```diff
@@ -72,8 +72,12 @@ final class StorePickerViewModel {
 // MARK: - Private helpers
 private extension StorePickerViewModel {
     func refetchSitesAndUpdateState() {
-        try? resultsController.performFetch()
-        state = StorePickerState(sites: resultsController.fetchedObjects)
+        do {
+            try resultsController.performFetch()
+            state = StorePickerState(sites: resultsController.fetchedObjects)
+        } catch {
+            DDLogError("Unable to re-fetch sites and update state: \(error)")
```

Comment:
Let's keep it consisten to add "⛔️" for error logs to make them stand out more in the console log:
```suggestion
            DDLogError("⛔️ Unable to re-fetch sites and update state: \(error)")
```